chirp/drivers/baofeng_wp970i.py

```diff
@@ -940,6 +966,9 @@ class BFA58S(WP970I):
     VENDOR = "Baofeng"
     MODEL = "BF-A58S"
     LENGTH_NAME = 7
+    POWER_LEVELS = [chirp_common.PowerLevel("High", watts=5.00),
+                    chirp_common.PowerLevel("---", watts=1.01),
```

Comment:
Erm, does this get exposed in this driver's valid_power_levels? If so, I don't think we should do this.

Reply:
OK. I though that was a trick you told me about before. I must have been thinking about something else. Is there a way to suppress the middle choice or a better way to do this? It hasn't been a problem for the other Baofengs because the the radios with 2 TX power levels use a different memory location that used for radios with 3 TX power levels. For some reason in this radio they use 0x00 for High and 0x02 for Low.

Reply:
Well, if you need to index into the list, making sure it always has enough entries is good, but not if some of those are invalid. Especially if there are enough bits to possibly mean this could get returned if memory was corrupted, that would be counter-productive. I haven't read through all of this driver, but why does this need to be three elements long? Are low and high 0 and 2 in memory? If it's possible that corrupted memory could select the middle entry, then I think it needs to be valid (perhaps just a duplicated Low entry). But, you'd want to filter that out in `get_features()`...

Reply:
I'm good with setting to be High/Low/Low. I think there is a driver I can look at that has High/Mid/Low and Mid in the radio doesn't work so the driver forces it to High (or maybe Low, I don't remember). I can do that tomorrow.